rasa_nlu/server.py

```diff
@@ -443,6 +458,16 @@ def unload_model(self, request):
             pre_load = router.project_store.keys()
         router._pre_load(pre_load)
 
+    if pre_load_model:
```

Comment:
Okay ! 

Do you mean using `project/all` to load all models from a certain project ? How would we deal with people naming their project `all` ? 

Otherwise, I would propose allowing regex in the model name like following

`--pre_load_model projectA/*`

What do you think ?

Reply:
It's a great idea, but ideally we would want the software to discern this sort of thing itself without the user prompting it.

You could add some kind of check in the user-specified directory - for instance, if it is a model, we'd expect a `metadata.json` file in there.